docs/pages/setup/reference/cli.mdx

```diff
@@ -1310,6 +1310,346 @@ $ tctl auth rotate --type=user --grace-period=200h
+  - members of `developer` group will receive `access` role.
+
+The values for `--secret`, `--id` and `--issuer-url` are provided by IdP.
+
+```code
+$ tctl sso configure oidc -n myauth -r groups,admin,access,editor,auditor -r group,developer,access \
+      --secret IDP_SECRET --id CLIENT_ID                                                            \
+      --issuer-url https://idp.example.com
+```
+
+2. Generate OIDC auth connector with Okta preset, enabled `groups` scope, mapping group `okta-admin` to roles `access`, `editor`, `auditor`.
+Issuer URL is set to match custom Okta domain.
+
+```code
+$ tctl sso configure oidc --preset okta --scope groups -r groups,okta-admin,access,editor,auditor \
+      --secret IDP_SECRET --id CLIENT_ID                                                          \
+      --issuer-url dev-123456.oktapreview.com
+```
+
+3. Generate OIDC auth connector with Google preset. Service account credentials are set to be loaded from /var/lib/teleport/gacc.json with --google-acc-uri.
+
+```code
+$ tctl sso configure oidc --preset google -r groups,mygroup@mydomain.example.com,access \
+      --secret SECRET --google-id GOOGLE_ID --google-acc-uri /var/lib/teleport/gacc.json \
+      --google-admin admin@mydomain.example.com
+```
+
+4. Generate the configuration and immediately test it using `tctl sso test`` command.
+
+```code
+$ tctl sso configure oidc ... | tctl sso test
+```
+
+
+### tctl sso configure saml
+
+Configure SAML auth connector, optionally using a preset.
+
+The flag `--attributes-to-roles/-r` can be provided multiple times. 
+
+To avoid errors when pasting XML to YAML file, we encourage the usage of flag `--entity-descriptor`/`-e`.
+
+```code
+$ tctl sso configure saml -e entity_desc.xml -r attr,value,role1 [<other-flags>]
+```
+
+#### Arguments
+
+This command accepts no arguments.
+
+#### Flags
+
+Mandatory flags: 
+- `--attributes-to-roles`; may be given multiple times.
+- `--entity-descriptor`, unless `--sso`, `--acs`, `--cert` and `--issuer` are also given.
+- `-n/--name`, unless `-p/--preset` is present.
+
+| Name                        | Description                                                                                                                                      |
+|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| `-p/--preset`               | Preset. One of: `okta`, `onelogin`, `ad`, `adfs`.                                                                                                |
+| `-n/--name`                 | Connector name. Required, unless implied from preset.                                                                                            |
+| `-e/--entity-descriptor`    | Set the Entity Descriptor. Valid values: file, URL, XML content. Supplies configuration parameters as single XML instead of individual elements. |
```

Comment:
```suggestion
| `-e/--entity-descriptor`    | Set the Entity Descriptor. Valid values: file path, URL, XML content. Supplies configuration parameters as a single XML document instead of individual elements. |
```